src/anyon/qpu_interface.jl

```diff
@@ -384,6 +397,20 @@ function read_response_body(body::Vector{UInt8})::String
     return body_string
 end
 
+function transpile_and_run_job(
+    qpu::AbstractQPU, 
+    circuit::QuantumCircuit,
+    num_repetitions::Integer;
+    transpiler::Transpiler=TrivialTranspiler()
```

Comment:
Remove the transpiler argument and use `get_transpiler(qpu)`. This is meant as a user-friendly "run", but without worrying about transpilers. If the user wants to use their own transpiler then they can call transpile themselves.